hepcrawl/parsers/elsevier.py

```diff
@@ -0,0 +1,615 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of hepcrawl.
+# Copyright (C) 2015, 2016, 2017 CERN.
+#
+# hepcrawl is a free software; you can redistribute it and/or modify it
+# under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+
+from __future__ import absolute_import, division, print_function
+
+import itertools
+import re
+from itertools import chain
+
+import six
+
+from inspire_schemas.api import LiteratureBuilder, ReferenceBuilder
+from inspire_schemas.utils import split_page_artid
+from inspire_utils.date import PartialDate
+from inspire_utils.helpers import maybe_int, remove_tags
+
+from ..utils import get_first, get_node
+
+
+DOCTYPE_MAPPING = {
+            'abs': 'abstract',
+            'add': 'addendum',
+            'adv': 'advertisement',
+            'ann': 'announcement',
+            'brv': 'book-review',
+            'cal': 'calendar',
+            'chp': 'chapter in a book',
+            'cnf': 'conference',
+            'con': 'contents list',
+            'cop': 'copyright information',
+            'cor': 'correspondence',
+            'crp': '',
+            'dis': 'discussion',
+            'dup': 'duplicate',
+            'edb': 'editorial board',
+            'edi': 'editorial',
+            'err': 'erratum',
+            'exm': 'exam',
+            'fla': 'full-length article',
+            'ind': 'index',
+            'lit': 'literature alert',
+            'mis': 'miscellaneous',
+            'nws': 'news',
+            'ocn': 'other contents',
+            'pgl': 'practice guidelines',
+            'pnt': 'patent report',
+            'prp': 'personal report',
+            'prv': 'product review',
+            'pub': 'publisher\'s Note',
+            'rem': 'removal',
+            'req': 'request for assistance',
+            'ret': 'retraction',
+            'rev': 'review',
+            'sco': 'short communication',
+            'ssu': 'short survey',
+        }
```

Comment:
where are these coming from? it seems to be different from https://www.elsevier.com/__data/assets/pdf_file/0010/784522/ja5_tagbytag5-v1.9.6.3.pdf page 52. Please use the list of abbreviations and names there.